hero tablet

DorotaDomanska opened the pull request.

DamianSikorski committed:
Dodanie animacji

DamianSikorski committed:
Animacja gallery

DamianSikorski committed:
Animacja gallery

DamianSikorski committed:
małe modyfikacje

DorotaDomanska committed:
hero tablet

DorotaDomanska committed:
hero tablet

DorotaDomanska committed:
hero tablet

DorotaDomanska committed:
hero tablet

DorotaDomanska committed:
hero tablet

DorotaDomanska committed:
hero tablet

DorotaDomanska committed:
hero tablet

DorotaDomanska committed:
hero tablet

kwiatkowsko commented:
do poprawy obwódka mobile, ::aftery w tablecie, dobre svg do podpięcia z następnym fixem

DorotaDomanska committed:
Dodanie hovera dla ciemnego buttona

DorotaDomanska committed:
Dodanie hovera dla ciemnego buttona

kwiatkowsko committed:
fix pin js-mobile-menu

kwiatkowsko committed:
Merge branch 'bugfix/DorotaDomanska/setup-button-hover' of https://github.com/KitsuneAkvma/tankteam-project-1 into bugfix/DorotaDomanska/setup-button-hover

kwiatkowsko committed:
Merge pull request #42 from KitsuneAkvma/bugfix/DorotaDomanska/setup-button-hover

Bugfix/dorota domanska/setup button hover

KitsuneAkvma committed:
mobile-menu/bugfix/BodyScrolllock error

Wywalono blokadę scrollbara ze względu na to, że w niczym nie przeszkadza przewijanie strony, zwłaszcza że pewna jej częśc i tak jest widoczna

kwiatkowsko committed:
Merge pull request #47 from KitsuneAkvma/mobile-menu/bugfix/bodyscrolllock-error

mobile-menu.js bugfix - BodyScrolllock deteled

DamianSikorski committed:
Animacja gallery

DamianSikorski committed:
Animacja gallery

DamianSikorski committed:
Animacja gallery

DamianSikorski committed:
Update _advantages.scss

kwiatkowsko committed:
Merge pull request #50 from KitsuneAkvma/feature/advantages-animation

Feature/advantages animation

kwiatkowsko committed:
Merge branch 'feature/gallery-animation' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/gallery-animation

DamianSikorski committed:
Update icons.svg

DamianSikorski committed:
Update icons.svg

DamianSikorski committed:
Update icons.svg

DamianSikorski committed:
Merge branch 'fix/Damian/icon-chevron' of https://github.com/KitsuneAkvma/tankteam-project-1 into fix/Damian/icon-chevron

kwiatkowsko committed:
Merge branch 'feature/gallery-animation' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/gallery-animation

kwiatkowsko committed:
Merge pull request #49 from KitsuneAkvma/feature/gallery-animation

Feature/gallery animation

DamianSikorski committed:
Update icons.svg

kwiatkowsko committed:
Merge branch 'fix/Damian/icon-chevron' of https://github.com/KitsuneAkvma/tankteam-project-1 into fix/Damian/icon-chevron

kwiatkowsko committed:
Merge pull request #51 from KitsuneAkvma/fix/Damian/icon-chevron

Fix/damian/icon chevron

DamianSikorski committed:
Lekka modyfikacja

aby nie uciekały przy ponownym scrollu do góry

DamianSikorski committed:
Lekka modyfikacja

aby nie uciekały przy ponownym scrollu do góry

DamianSikorski committed:
Merge branch 'feature/advantages-animation' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/advantages-animation

DamianSikorski committed:
Update gallery-animation.js

DamianSikorski committed:
Update gallery-animation.js

DamianSikorski committed:
Merge branch 'feature/gallery-animation' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/gallery-animation

KitsuneAkvma reviewed (approved):
Kod wygląda w porządku niestety nie mam jak odpalić podglądu. Zmerguje dopiero po północy

kwiatkowsko committed:
Merge pull request #53 from KitsuneAkvma/feature/gallery-animation

Feature/gallery animation

kwiatkowsko committed:
Merge pull request #52 from KitsuneAkvma/feature/advantages-animation

Feature/advantages animation

kwiatkowsko committed:
zmiana

kwiatkowsko commented:
pozostaje fix buttonów i stylizacja pod nowe svg i będzie śmigać :)

kwiatkowsko reviewed (changes requested):
Poprawki

DorotaDomanska committed:
dodanie chevron

kwiatkowsko committed:
Merge branch 'feature/DorotaDomanska/tablet' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/DorotaDomanska/tablet
